src/util/checkWalletForNetwork.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { InjectedWindow } from '@polkadot/extension-inject/types';
+import { Wallet } from '~src/types';
+
+export const checkWalletForNetwork = (network: string) => {
+	if(!network) return;
+	const isNovaWallet = (window as any).walletExtension?.isNovaWallet;
+	const isPolymeshWallet = ['polymesh'].includes(network);
+	const injectedWindow = window as Window & InjectedWindow;
+	const availableWallets = injectedWindow.injectedWeb3;
+
+	if(!isPolymeshWallet && !isNovaWallet){
```

Comment:
`!isNovaWallet` check should be removed and should be  inside isPolymeshWallet check code block